html-vzor/style.css

```diff
@@ -110,7 +110,7 @@ h1, h2, h3, h4, h5, h6 {
 	border-bottom: 1px solid #ba6e46;
 }
 
-.dresses .product__image {
+.dresses .productImage {
 	aspect-ratio: 3 / 4;

```

Comment:
ty změny v html_vzor/xxx nic nedělají a technicky bys neměla měnit "vzor", ale jen abys věděla, nic to nerozbíjí